### DISCUSSION TITLE
📢 Choose our framework !

### DISCUSSION THREAD

**Discussion by mkubdev** [WebXDAO/WebXDAO.github.io#37] [Ideas]
## 👋 Welcome Web3Community folks!

📢 **Choose our framework!**

  We're in the process of planning the integration of a framework for our website. Knowing that you must come from different backgrounds, we want to know if you would have a preference on choosing this framework :
  * [Next.js](https://nextjs.org/) 
  * [Angular](https://angular.io/)
  * [Vue.js](https://vuejs.org/)
  
  We need you :
  * To talk about this subject.
  * Share ideas.
  * Engage with other community members.
  *  Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  
  To get started, comment below with your ideas and preferences, as well as detail why this is your favorite framework.

  🔗 If you want to join the organization, you can use this issue templates : **[Invite me to the Web 3 Community](https://github.com/web3community/start-here/issues/new?assignees=&labels=github-invitation&template=invitation.yml&title=Please+invite+me+to+the+community)**. If you need help, here's a [link to the documentation](https://github.com/web3community/start-here).

**Comment by suitupstan** [comment]
Astro + Vue would be cool

**Reply by suitupstan** [reply]
We've seen a example implementation as well now: https://github.com/web3community/devprotocol.xyz

**Comment by Bryson69** [comment]
NextJs

**Reply by Panquesito7** [reply]
I like this! NextJS is very cool and works very well. I've used it on a website I maintain. 🙂

**Comment by vinzvinci** [comment]
Next.js and Tailwind ✅
